### ISSUE TITLE
Widget breaks when compiling using npm start

### ISSUE THREAD

**Issue by mchmx** [mendixlabs/list-view-controls#18]
After following the contribution instructions as described, the compiled .mpk is not working properly. Upon inspecting its contents, none of the expected xml files are present. I don't know if this is the root cause, but the modeler is then unable to interpret the widget and the widget is not functional anymore.

**Comment by Andries-Smit** [comment]
Think there is an issue in the start command, remove "\  from the https://github.com/mendixlabs/list-view-controls/blob/master/package.json#L8
``` js
 "start": "nodemon --watch webpack.config.js --watch Gruntfile.js --exec grunt",
```
you can also start grunt directly typing:
> grunt

**Comment by Andries-Smit** [comment]
Fixed and release  v1.2.1 
https://appstore.home.mendix.com/link/app/105694/
